alien/CMakeLists.txt

```diff
@@ -56,18 +56,33 @@ arcane_framework_set_if_unset(Arctools_ROOT ${ARCFRAMEWORK_ROOT}/arctools PATH "
 
 list(APPEND CMAKE_MODULE_PATH ${ARCFRAMEWORK_ROOT}/_common/build_all/cmake)
 
-# Add Alien only if blas and boost found
+#first try to look for MKL
+message(STATUS "Looking for mkl BLAS")
+set(BLA_VENDOR Intel10_64lp_seq)
 find_package(BLAS)
-find_package(Boost OPTIONAL_COMPONENTS program_options)
-if (BLAS_FOUND AND Boost_program_options_FOUND)
-    add_subdirectory(standalone)
-    set(ALIEN_FOUND YES)
-    set(FRAMEWORK_INSTALL YES)
-    if (WIN32) # Not yet ready...
-#        if (BUILD_ARCANE_INTERFACE)
-#            add_subdirectory(ArcaneInterface)
-#        endif ()
-    elseif (BUILD_ARCANE_INTERFACE)
-            add_subdirectory(ArcaneInterface)
-    endif ()
+if(BLAS_FOUND)
+    add_definitions(-DALIEN_USE_MKL)
 endif()
+
+if(NOT BLAS_FOUND)
+    message(STATUS "Looking for OpenBLAS BLAS")
+    set(BLA_VENDOR OpenBLAS)
+    find_package(BLAS REQUIRED)
+    set(CBLAS_INCLUDE_DIR CBLAS_INCLUDE_DIR-NOTFOUND)
+    find_path(CBLAS_INCLUDE_DIR cblas.h HINTS ENV OPENBLAS_ROOT PATH_SUFFIXES openblas include REQUIRED)
+    message(STATUS "cblas.h found in ${CBLAS_INCLUDE_DIR}")
+    include_directories(${CBLAS_INCLUDE_DIR})
```

Comment:
Do not add to global include path for all the targets.
CMake define a target for BLAS. We should have to use it:
```
target_link_libraries(my_target PUBLIC BLAS::BLAS)
```

I think it may fix the CI on ubuntu